### ISSUE TITLE
how many epoches need by RNN_SPSS?

### ISSUE THREAD

**Issue by jennywx2020** [ZackHodari/morgana#2]
I have tried RNN_SNSS using blizzard2017, but after 100 epoches, the wav seems can't synthesized. and the loss and distortion of mcep and bap is not changed much. I wonder maybe more epoches are needed.

**Comment by ZackHodari** [comment]
Sorry, I shouldn't have uploaded that model as I haven't properly tested it. My intention was to provide a basic re-implementation of the standard [Merlin](https://github.com/CSTR-Edinburgh/merlin/) architecture.

The methods in `RNN_SPSS` should function correctly: data+normalisers; prediction w/MLPG; loss; and synthesis during validation+testing. However, the model hyper parameters might need tuning (layers, batch size, learning rate, epochs, etc.) If you get it working please share what you tried, but the right settings might also change for different datasets.

When I get chance I plan to create recipes for data-prep and training for a few standard models (e.g. SPSS, Tacotron), but that will take time

**Comment by jennywx2020** [comment]
OK. I will try tuning it and compare the model with merlin to see if it works. BTW, the Morgana architecture is very clear and goof for reuse.
